package.json

```diff
@@ -176,4 +176,4 @@
     "**/node-fetch": "^2.6.7",
     "**/markdown-it": "^12.3.2"
   }
-}
+}
```

Comment:
supernit: please bring back the trailing newline

Reply:
will do now